### ISSUE TITLE
Submit PubMode to the MELPA repository?

### ISSUE THREAD

**Issue by Guillawme** [jshoyer/pubmode#2]
Hi there,

I initially posted this issue [on the original repository](https://github.com/wash/pubmode/issues/4), before I figured out the project was dormant and continued here.

Would you mind submitting PubMode to the MELPA repository? That would make it discoverable and installable directly from within Emacs' `package.el`.

See here for how to submit (that's basically simply a PR on GitHub, once the package fits their guidelines): https://github.com/melpa/melpa/blob/master/CONTRIBUTING.md

**Comment by jshoyer** [comment]
Submitting to MELPA sounds like a good idea. I do not really use `package.el`, so it had not occurred to me.

I am not sure if they will accept my fork though, because Stefan's version is the "official" repository, inasmuch as there is one. My modifications are pretty minor.

Have you created a MELPA recipe before?

**Comment by Guillawme** [comment]
No, I've never submitted anything to MELPA. Writing the recipe and submitting seem easy and I can help you, but I cannot help with getting the package to comply with their many guidelines (I have a very limited programming experience, and know virtually nothing about Emacs Lisp except very simple init file configuration).

I'm glad you're interested. :-)
